README.md

```diff
@@ -62,14 +62,14 @@ Create a new Volto project by using the `@plone/generator-volto` utility.
 
 It will bootstrap a Volto project in a folder of your choice with all the required
 boilerplate to start customizing your Volto site.
-
-    $ npm install -g yo @plone/generator-volto
-    $ yo @plone/volto
-
+```
+sudo npm install -g yo @plone/generator-volto
```

Comment:
`sudo` should not be required to install global dependencies. See https://stackoverflow.com/a/20264272/2214933 and https://medium.com/@ExplosionPills/dont-use-sudo-with-npm-5711d2726aa3. If you must use it, then something is not right with your system configuration. Nonetheless, we need to document what that something is. I don't know what that is at this moment.

Reply:
@stevepiercy Thank you for taking the time to review this PR; I've made some modifications now and would appreciate it if you could have a look at it.